### PR TITLE
fix(csp): make CSP reports more detailed, remove sample rate

### LINKED ISSUE #3297: CSP report only updates
We should add these:

- [ ] Add script sample
- [ ] UA String 

Ref:
https://kibana.fxa.us-west-2.prod.mozaws.net/#/dashboard/elasticsearch/CSP%20Violations

### PR DESCRIPTION
**Pull request by vladikoff**
Fix #3297

### CODE REVIEW

**Comment by vladikoff** [comment]
@jrgm this removes the 10% sample rate and makes it 100% for all csp errors, also adds a few extra fields that you requested. let me know what you think

File: server/lib/routes/post-csp.js [hunk 1]
```diff
@@ -42,11 +24,13 @@ module.exports = function (options) {
       var report = req.body['csp-report'];
 
       var entry = {
+        agent: req.get('User-Agent'),
```
Comment (vladikoff) [inline server/lib/routes/post-csp.js]: @jrgm what should be the name of this?

File: server/lib/routes/post-csp.js [hunk 2]
```diff
@@ -42,11 +24,13 @@ module.exports = function (options) {
...
         blocked: report['blocked-uri'],
         op: 'server.csp',
         referrer: report['referrer'],
+        source: report['source-file'],
```
Comment (vladikoff) [inline server/lib/routes/post-csp.js]: From @jrgm , let's add `script sample` (sp?) just to see if it logs

Comment (vladikoff) [inline server/lib/routes/post-csp.js]: Added!

**Comment by vladikoff** [comment]
Updated @jrgm r?

See https://github.com/mozilla/fxa-content-server/pull/3347#discussion_r48663762

File: server/lib/routes/post-csp.js (hunk 1 above)
Comment (jrgm) [inline server/lib/routes/post-csp.js]: 'agent' as the key name is the correct name. Merge away.

**Comment by jrgm** [comment]
r+

**Comment by vladikoff** [comment]
@jrgm Thank you!

**Pull request merged**